### PR TITLE
ADEN-7933 Adding stickyAd events

### PR DESCRIPTION
**Pull request by dmnsobczak**

### CODE REVIEW

File: app/modules/ads/tracking/slot-tracker.js [hunk 1]
```diff
@@ -6,10 +6,12 @@ const onRenderEndedStatusToTrack = [
   'success',
 ];
 const onChangeStatusToTrack = [
+  'success',
```
Comment (fraszczakszymon) [inline app/modules/ads/tracking/slot-tracker.js]: It will cause multiple events in DW for single slot

Comment (dmnsobczak) [reply app/modules/ads/tracking/slot-tracker.js]: Ack, reverting.

**Review by fraszczakszymon — APPROVED** [review]

File: app/modules/ads/tracking/slot-tracker.js [hunk 2]
```diff
@@ -10,8 +10,6 @@ const onChangeStatusToTrack = [
   'catlapsed',
   'error',
   'viewport-conflict',
```
Comment (jacekbj) [inline app/modules/ads/tracking/slot-tracker.js:12]: `'viewport-conflict'` has also been removed from status.
https://github.com/Wikia/ad-engine/pull/221/files#diff-3755f3ded23b967a2b9a6ae83dee2bb8L124

**Review by jacekbj — CHANGES_REQUESTED** [review]

File: app/modules/ads/tracking/slot-tracker.js (hunk 2 above)
Comment (jacekbj) [reply app/modules/ads/tracking/slot-tracker.js:12]: I'm not really sure if it needs a change.

**Review by jacekbj — APPROVED** [review]

File: app/modules/ads/tracking/slot-tracker.js (hunk 2 above)
Comment (dmnsobczak) [reply app/modules/ads/tracking/slot-tracker.js:12]: Slot can be disabled with `viewport-conflict` status, which is equal to `collapse`:
https://github.com/Wikia/ad-engine/blob/dev/src/ad-engine/services/btf-blocker-service.js#L73

**Pull request merged**